RELEASE.md

```diff
@@ -30,6 +30,7 @@
 * Fixed bug causing the `after_dataset_saved` hook only to be called for one output dataset when multiple are saved in a single node and async saving is in use.
 * Log level for "Credentials not found in your Kedro project config" was changed from `WARNING` to `DEBUG`.
 * Added safe extraction of tar files in `micropkg pull` to fix vulnerability caused by [CVE-2007-4559](https://github.com/advisories/GHSA-gw9q-c7gh-j9vm).
+* Fixed bug that didn't allow to read or write datasets with `s3a` or `s3n` filepaths
```

Comment:
We've just released `0.18.5`. Would you mind moving this to `0.18.6`?

Reply:
Done @merelcht !
Would appreciate if 0.18.6 is released soon 😊

You merge or I do?

Reply:
We'll do our best!! I'll merge it now. Thanks again ⭐